lightning/mydump/parser_test.go

```diff
@@ -19,11 +19,12 @@ import (
 	"strings"
 
 	. "github.com/pingcap/check"
+	"github.com/pingcap/errors"
```

Comment:
we don't add more test for it, only old cases

Reply:
The old unit test didn't compile since the interface is changed. There's no new tests in this file.